### PR TITLE
raidboss: e12s oracle cataclysm callout

### PR DESCRIPTION
**Pull request by Legends0**
Call out opposite direction of cataclysm, a bit more advanced than "getBehind".

I had tested getBehind early on in the tier, but never made a PR with it as ideally a cardinal would have been nicer to have. I will test this out later this week, I am unsure if delaySeconds 0.5 is enough.

### CODE REVIEW

File: ui/raidboss/data/05-shb/raid/e12s.js [hunk 1]
```diff
@@ -978,6 +978,81 @@ export default {
...
+          data.safeZone = null;
+          return;
+        }
+        if (!combatantData.combatants) {
+          console.error(`Oracle Of Darkness: null combatants`);
+          data.safeZone = null;
+          return;
+        }
+
+        // we need to filter for the Oracle Of Darkness with the highest ID
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/raid/e12s.js:1021]: Is it the same id as the one who is doing this cast such that you could use `matches.sourceId`?

Comment (valarnin) [reply ui/raidboss/data/05-shb/raid/e12s.js:1021]: Looks like that's the case to me at least, though I'd need to test it to be sure:
```
20|2021-02-13T22:37:29.2990000-06:00|4001ED5E|Oracle Of Darkness|58C2|Cataclysm|E0000000||5.70||e1c6cf0d8bbd16afba599ca69c645583
```

As such the code above could probably be:
```js
        const combatantData = await window.callOverlayHandler({
          call: 'getCombatants',
          ids: [matches.sourceId],
        });
```

Comment (Legends0) [reply ui/raidboss/data/05-shb/raid/e12s.js]: I didn't realize it, but yea I can use ids like so to get the correct combatant. Thanks.

**Comment by quisquous** [comment]
Also, the original patch worked great for me tonight when I tested it.

**Pull request merged**